### DISCUSSION TITLE
Add individual parts?

### DISCUSSION THREAD

**Discussion by lolobala86** [rgriebl/brickstore#875] [Q&A] [answered]
I can use "Add parts" to search for both individual parts and sets. For sets, the Price Guide is displayed and I can also add them to a document. This does not work for individual parts. The Price Guide data is not displayed and the "Add" field is grayed out.

I haven't found any information about this anywhere yet, it seems to work normally for everyone, but not for me.

Can you perhaps help me? I'm probably just too stupid and have set something wrong. But I'm getting desperate...

I am using Brickstore 2024.4.1 on macOS 14.4.1

Thank you very much!

**Comment by rgriebl** [comment, marked as answer]
Parts are colored, so you just need to pick one.

**Reply by lolobala86** [reply]
This is how it works! So I really was that stupid... Thanks!
